example

```diff
@@ -0,0 +1 @@
+>>> p = subprocess.run(["fzf", '-f', 'b'], stdout=subprocess.PIPE, input="baz\nbiz\nfoo\nbar", encoding='UTF-8')
```

Comment:
This was probably added in 0237172e2ca0c7ccfbae8ca30cebd136fd8fadda by accident?